pkg/api/server/v1alpha2/log/gcs_test.go

```diff
@@ -79,6 +79,7 @@ func NewRecordReplayClient(t *testing.T, modReq func(r *httpreplay.Recorder), po
 	if err != nil {
 		t.Fatal(err)
 	}
+	rep.IgnoreHeader("X-Goog-Gcs-Idempotency-Token")
```

Comment:
why do we ignore this one?

Reply:
This unit tests uses http replay. Earlier, when we ran this on a live cluster, GCS sdk supported this differently. Now, this header gives us error, to resolve this either we ignore this header or run against live GCS bucket and regenerate test data.